db/c.cc

```diff
@@ -1077,6 +1078,43 @@ void rocksdb_writebatch_put_cf(
   b->rep.Put(column_family->rep, Slice(key, klen), Slice(val, vlen));
 }
 
+void rocksdb_writebatch_putv(
+    rocksdb_writebatch_t* b,
+    size_t num_keys, const char* const* keys_list,
+    const size_t* keys_list_sizes,
+    size_t num_values, const char* const* values_list,
+    const size_t* values_list_sizes) {
+  Slice key_slices[num_keys];
```

Comment:
Ah, this fails the commit with 

    10:26:28 db/c.cc:1087:19: error: variable length array of non-POD element type 'rocksdb::Slice'


I have to revert this, sorry :(